### PR TITLE
fix: cross compile binary with CGO to satisfy sqlite3

### PR DESCRIPTION
**Pull request by hoshsadiq**
**Requirements**

- [ ] I have added test coverage for new or changed functionality
- [ ] I have followed the repository's [pull request submission guidelines](../blob/main/CONTRIBUTING.md#submitting-pull-requests)
- [ ] I have validated my changes against all supported platform versions

**Related issues**

Provide links to any issues in this repository or elsewhere relating to this pull request.

**Describe the solution you've provided**

Provide a clear and concise description of what you expect to happen.

**Describe alternatives you've considered**

Provide a clear and concise description of any alternative solutions or features you've considered.

**Additional context**

Add any other context about the pull request here.

### CODE REVIEW

**Review by mike-zorn — APPROVED** [review]
AMAZING!!!!!!

❤️ ❤️ ❤️ ❤️ ❤️ ❤️ ❤️ ❤️ ❤️ ❤️ ❤️ ❤️ ❤️ ❤️ ❤️ ❤️ ❤️ ❤️ ❤️ ❤️ ❤️ ❤️ ❤️ ❤️ ❤️ ❤️ ❤️ ❤️ ❤️ ❤️ ❤️ ❤️ ❤️ ❤️ ❤️ ❤️ ❤️ ❤️ ❤️ ❤️ ❤️ ❤️ ❤️ ❤️ ❤️ ❤️ ❤️ ❤️ ❤️ ❤️ ❤️ ❤️ ❤️ ❤️ ❤️ ❤️

File: .github/actions/mike-test-build/action.yml [hunk 1]
```diff
@@ -0,0 +1,60 @@
+name: Publish Artifacts
```
Comment (mike-zorn) [inline .github/actions/mike-test-build/action.yml:1]: Probably we should remove the test actions?

Comment (hoshsadiq) [reply .github/actions/mike-test-build/action.yml:1]: I guess I wasn't sure what the aim was for this action. I figured you're trying to potentially get the builds running for the temp branch? The trick is the docker run step down below. I'll have to replace the other action if we want to get rid of it.

File: .github/actions/mike-test-build/action.yml [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+    - uses: hoshsadiq/ssh2action@main
+
+    - shell: bash
+      run: docker exec \
+        --env GITHUB_TOKEN \
+        --env HOMEBREW_DEPLOY_KEY \
+        --workdir "$PWD" \
+        --tty \
+        "$CONTAINER_ID" \
+        goreleaser build --snapshot --config .goreleaser.yaml
```
Comment (BehnH) [inline .github/actions/mike-test-build/action.yml:60]:
```suggestion
      run: >
        docker exec 
        --env GITHUB_TOKEN 
        --env HOMEBREW_DEPLOY_KEY 
        --workdir "$PWD" 
        --tty 
        "$CONTAINER_ID" 
        goreleaser build --snapshot --config .goreleaser.yaml
```

**Pull request merged**